build fails with `write jobserver: Bad file descriptor`

```
13.90 [ERROR] The compilation of coq-dpdgraph.1.0+9.1 failed at "make -j10 WARN_ERR=".
13.90 
13.90 #=== ERROR while compiling coq-dpdgraph.1.0+9.1 ===============================#
13.90 # context              2.3.0 | linux/x86_64 | ocaml-base-compiler.4.14.2 | https://coq.inria.fr/opam/released#2025-12-10 02:47
13.90 # path                 ~/.opam/ocaml-4.14.2/.opam-switch/build/coq-dpdgraph.1.0+9.1
13.90 # command              /usr/bin/make -j10 WARN_ERR=
13.90 # exit-code            2
13.90 # env-file             ~/.opam/log/coq-dpdgraph-7-2e284d.env
13.90 # output-file          ~/.opam/log/coq-dpdgraph-7-2e284d.out
13.90 ### output ###
13.90 # [...]
13.90 # ocamlfind opt -verbose -package ocamlgraph -linkpkg -o dpdusage version.mli dpd_compute.mli dpd_dot.mli dpd_parse.mli dpd_lex.mli version.ml dpd_compute.ml dpd_dot.ml dpd_parse.ml  dpd_lex.ml \
13.90 #          dpdusage.mli dpdusage.ml
13.90 # echo "let version = \"""\"" >> version.ml
13.90 # make[2]: *** write jobserver: Bad file descriptor.  Stop.
13.90 # make[2]: *** Waiting for unfinished jobs....
13.90 # make[2]: *** write jobserver: Bad file descriptor.  Stop.
13.90 # make[1]: *** [Make_coq:413: all] Error 2
13.90 # make[1]: Leaving directory '/root/.opam/ocaml-4.14.2/.opam-switch/build/coq-dpdgraph.1.0+9.1'
13.90 # 29 states, 495 transitions, table size 2154 bytes
13.90 # make: *** [Makefile:15: all] Error 2
13.90 # Effective set of compiler predicates: pkg_ocamlgraph,autolink,native
13.90 # + ocamlopt.opt -verbose -o dpdusage -I /root/.opam/ocaml-4.14.2/lib/ocamlgraph /root/.opam/ocaml-4.14.2/lib/ocamlgraph/graph.cmxa version.mli dpd_compute.mli dpd_dot.mli dpd_parse.mli dpd_lex.mli version.ml dpd_compute.ml dpd_dot.ml dpd_parse.ml dpd_lex.ml dpdusage.mli dpdusage.ml
```
I'm guessing this is because we use `make` instead of `$(MAKE)` at https://github.com/rocq-community/coq-dpdgraph/blob/d7f77a21edb23c52e63ce1f163ddc35962c5e9f0/Makefile#L15 , but I'm not sure

I am making it from the dockerfile
```
# syntax=docker/dockerfile:1.4
FROM docker.io/ubuntu:22.04
ENV DEBIAN_FRONTEND=noninteractive
RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates && apt-get clean
RUN apt-get update && apt-get install -y software-properties-common curl sudo
RUN add-apt-repository ppa:deadsnakes/ppa
RUN apt-get update && apt-get install -y \
    curl \
    && add-apt-repository ppa:deadsnakes/ppa \
    && apt-get update \
    && apt-get install -y \
    python3.11 \
    python3.11-venv \
    python3.11-dev \
    python3-pip \
    wget \
    git \
    make \
    bubblewrap \
    unzip \
    libgmp-dev \
    pkg-config \
    && apt-get clean
RUN wget https://github.com/ocaml/opam/releases/download/2.3.0/opam-2.3.0-x86_64-linux -O /usr/local/bin/opam && \
    chmod +x /usr/local/bin/opam
RUN opam init --disable-sandboxing --yes --bare && \
    opam switch create ocaml-4.14.2 4.14.2 && \
    opam repo add coq-released https://coq.inria.fr/opam/released --all-switches
RUN echo 'eval $(opam env)' >> /etc/bash.bashrc
RUN opam install -y --confirm-level=unsafe-yes coq.9.1.0
RUN opam install -y --confirm-level=unsafe-yes coq-dpdgraph
```

--- a/Makefile
+++ b/Makefile
@@ -6,38 +6,10 @@
 #       GNU Lesser General Public License Version 2.1
 #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-NAME=coq-dpdgraph
-VERSION=1.0-9.0
-
 main_target : all
 
 all install uninstall test test-suite clean clean_test : Make_coq
-	make -f $< $@
+	$(MAKE) -f $< $@
 
 Make_coq : Make
 	rocq makefile -f $< -o $@
-
-#-------------------------------------------------------------------------------
-DISTRIBUTED+=Makefile LICENSE README.md configure Makefile.in
-
-distrib : $(NAME)-$(VERSION).tgz
-
-%.tgz : clean
-	$(ECHO_CIBLE)
-	rm -rf $* $@
-	mkdir $*
-	cp --parents $(DISTRIBUTED) $*
-	tar zcvf $@ $*
-	rm -rf $*
-	$(ECHO) "Don't forget to copy README.md and $@ on the server if needed"
-
-
-#-------------------------------------------------------------------------------
-# testing
-
-
-#-------------------------------------------------------------------------------
-clean_coq : Make_coq
-	$(MAKE) -f $< clean
-
-#-------------------------------------------------------------------------------

--- a/README.md
+++ b/README.md
@@ -258,6 +258,27 @@ In the example above it reports that ``Permutation_app_swap`` was
 referenced 0 times.  You can specify max number of references allowed
 (default 0) via ``-threshold`` command line option.
 
+## Making new releases for each version of Rocq
+
+  - The coq-master branch is maintained to compile with the current
+   version of the rocq prover master branch.  Rocq developers usually
+   take care of that.  When a version of this package needs to be produced
+   to be used in an opam package, it is usually necessary to find in the
+   history the closest commit to the date of the release of rocq that is
+   aimed for.
+
+  - Once this commit is found it is reasonable to mark this as a branch and
+  use it to produce a release on github.  This branch can be used to produce
+  a compressed tar archive as follows.
+
+  ```
+  git checkout -b <branch-name>
+  git archive --format tgz -o coq-dpdgraph-1.0-9.1.tgz
+  ```
+  The `tgz` file thus produced should be uploaded to a new release on github
+  and this file's url should be used in a new opam package description
+  in the opam archive.
+
 ## Development information
 
 ### Generated ``.dpd`` format description

--- a/meta.yml
+++ b/meta.yml
@@ -285,6 +285,27 @@ documentation: |
   referenced 0 times.  You can specify max number of references allowed
   (default 0) via ``-threshold`` command line option.
 
+  ## Making new releases for each version of Rocq
+
+    - The coq-master branch is maintained to compile with the current
+     version of the rocq prover master branch.  Rocq developers usually
+     take care of that.  When a version of this package needs to be produced
+     to be used in an opam package, it is usually necessary to find in the
+     history the closest commit to the date of the release of rocq that is
+     aimed for.
+
+    - Once this commit is found it is reasonable to mark this as a branch and
+    use it to produce a release on github.  This branch can be used to produce
+    a compressed tar archive as follows.
+
+    ```
+    git checkout -b <branch-name>
+    git archive --format tgz -o coq-dpdgraph-1.0-9.1.tgz
+    ```
+    The `tgz` file thus produced should be uploaded to a new release on github
+    and this file's url should be used in a new opam package description
+    in the opam archive.
+
   ## Development information
   
   ### Generated ``.dpd`` format description

